src/Admin/ElectedRepresentative/ElectedRepresentativeAdmin.php

```diff
@@ -451,6 +449,16 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                     }
                 },
             ])
+            ->add('userListDefinitions', ModelAutocompleteFilter::class, [
```

Comment:
do we need to check here the specific UserList type for avoiding to display referent user lists?